tests/testcases/030_check-network.yml

```diff
@@ -141,7 +162,7 @@
     - "{{ pod_ips }}"
 
   - name: Ping between hostnet pods is working
-    command: "{{ bin_dir }}/kubectl -n test exec {{ item[0] }} -- ping -c 4 {{ item[1] }}"
+    command: "{{ bin_dir }}/kubectl -n test exec {{ item[0] }} -- curl {{ item[1] }}:8080"
     when:
     - item[0] in pods_hostnet
     - item[1] in pods_hostnet
```

Comment:
This test file is passed successfully, but still facing another issue on another file:
```
TASK [Wait for netchecker server] **********************************************
task path: /builds/kargo-ci/kubernetes-sigs-kubespray/tests/testcases/040_check-network-adv.yml:32
Friday 14 October 2022  11:33:37 +0000 (0:00:00.414)       0:00:05.158 ******** 
FAILED - RETRYING: Wait for netchecker server (3 retries left).
FAILED - RETRYING: Wait for netchecker server (2 retries left).
FAILED - RETRYING: Wait for netchecker server (1 retries left).
fatal: [instance-1]: FAILED! => {"attempts": 3, "changed": true, "cmd": "set -o pipefail && /usr/local/bin/kubectl get pods -o wide --namespace default | grep ^netchecker-server", "delta": "0:00:00.073394", "end": "2022-10-14 11:34:08.470141", "msg": "non-zero return code", "rc": 1, "start": "2022-10-14 11:34:08.396747", "stderr": "No resources found in default namespace.", "stderr_lines": ["No resources found in default namespace."], "stdout": "", "stdout_lines": []}
```

Reply:
```
changed: [instance-1] => {"changed": true, "cmd": "/usr/local/bin/kubectl get deployment --namespace default", "delta": "0:00:00.075904", "end": "2022-10-17 00:42:34.826447", "msg": "", "rc": 0, "start": "2022-10-17 00:42:34.750543", "stderr": "", "stderr_lines": [], "stdout": "NAME                READY   UP-TO-DATE   AVAILABLE   AGE\nnetchecker-server   0/1     0            0           84s", "stdout_lines": ["NAME                READY   UP-TO-DATE   AVAILABLE   AGE", "netchecker-server   0/1     0            0           84s"]}
```

Reply:
```
TASK [Check connectivity with all netchecker agents] ***************************
task path: /builds/kargo-ci/kubernetes-sigs-kubespray/tests/testcases/040_check-network-adv.yml:162
Monday 17 October 2022  04:18:29 +0000 (0:00:00.098)       0:03:45.651 ******** 
fatal: [instance-1 -> instance-1]: FAILED! => {
    "assertion": "agents_check_result is defined",
    "changed": false,
    "evaluated_to": false,
    "msg": "Connectivity check to netchecker agents failed"
}
```